Tutorials/intro-to-QUAlibrate/calibrations/readme.md

```diff
@@ -0,0 +1,249 @@
+# 🔧 Convert a QUA Program into a QualibrationNode
+
+This guide walks you through converting the [`03_time_of_flight.py`](https://github.com/qua-platform/qua-libs/blob/main/Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/03_time_of_flight.py) QUA protocol into a modular `QualibrationNode` using QUAlibrate. The resulting node can be run via Python or through the QUAlibrate Web Interface.
```

Comment:
I think that having the two version of the script (the standard one and the one converted to a qualibration node) would help customers understand the difference, what do you think?